spec/factories/author.rb

```diff
@@ -32,4 +39,37 @@
       end
     end
   end
+
+  # Public data from
+  # - https://stanfordwho.stanford.edu
+  # - https://med.stanford.edu/profiles/russ-altman
+  factory :russ_altman, parent: :author do
```

Comment:
Does this mean `author.local.rb` can go away?

Reply:
It's all public information; if it's also in that local file, yea, will throw that out in this commit.

Reply:
Turns out the local version adds a ton of publications that are specific to SW.  Until we remove SW, punting on removing that data.  This PR allows the public information to be used generally in specs and CI.

Reply:
But now, the only thing we are keeping "private" is a bunch of ID numbers?  I'm not saying they need to be the same factory, but I don't see any privacy concern over a few hundred numerals.